src/content/ui-api/kubectl-gs/template-nodepool.md

```diff
@@ -55,6 +55,7 @@ Here are the supported flags:
 - `--nodes-min` - minimum number of worker nodes for the node pool. (default 3)
 - `--output` - Sets a file path to write the output to. If not set, standard output will be used.
 - `--organization` - Name of the organization owning the cluster.
+- `--release` - The parent workload cluster's Giant Swarm release version.
```

Comment:
"Parent" isn't needed. A node pool belongs to a workload cluster.

"Giant Swarm release" is not the term we use.

```suggestion
- `--release` - The workload cluster release version.
```